src/wp-admin/includes/upgrade.php

```diff
@@ -2247,30 +2551,31 @@ function dbDelta( $queries = '', $execute = true ) {
 		}
 
 		// Fetch the table column structure from the database
-		$suppress = $wpdb->suppress_errors();
-		$tablefields = $wpdb->get_results("DESCRIBE {$table};");
+		$suppress    = $wpdb->suppress_errors();
+		$tablefields = $wpdb->get_results( "DESCRIBE {$table};" );
 		$wpdb->suppress_errors( $suppress );
 
-		if ( ! $tablefields )
+		if ( ! $tablefields ) {
 			continue;
+		}
 
 		// Clear the field and index arrays.
 		$cfields = $indices = $indices_without_subparts = array();
 
 		// Get all of the field names in the query from between the parentheses.
-		preg_match("|\((.*)\)|ms", $qry, $match2);
-		$qryline = trim($match2[1]);
+		preg_match( '|\((.*)\)|ms', $qry, $match2 );
```

Comment:
Is this regular expression the same in single quotes?

Reply:
It seems it is single quotes upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/upgrade.php#L2816

Reply:
That wasn't the question. Run a little test to see if the two are equivalent. 
I think this one might be okay since it's not `\\`